noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -1,9 +1,17 @@
+// This file contains some unused functions that may be needed later (e.g. those filling top 6 bits of a 31-byte
+// field). Little endian functions are kept for reference but likely won't be used. Unused functions should be
+// removed before audit.
+
+pub mod array;
+pub mod bounded_vec;
+
 use crate::oracle::random::random;
 
-/// Converts the input bytes into an array of fields. A Field is ~254 bits meaning that each field can store 31 whole bytes.
+/// Converts the input bytes into an array of fields. A Field is ~254 bits meaning that each field can store 31 whole
+/// bytes.
```

Comment:
Line width of 120 chars was not followed here and it made this hard to read. Would be lovely if linter could scream here.